apps/ledger-live-mobile/src/components/Web3AppWebview/PlatformAPIWebview.tsx

```diff
@@ -504,6 +505,17 @@ export const PlatformAPIWebview = forwardRef<WebviewAPI, WebviewProps>(
       tracking.platformLoadFail(manifest);
     }, [manifest, tracking]);
 
+    const onOpenWindow = useCallback((event: WebViewOpenWindowEvent) => {
+      const { targetUrl } = event.nativeEvent;
+      Linking.canOpenURL(targetUrl).then(supported => {
+        if (supported) {
+          Linking.openURL(targetUrl);
+        } else {
+          console.error(`Don't know how to open URI: ${targetUrl}`);
```

Comment:
Might be a better error message ?

onOpenWindow | URL_UNSUPPORTED : Unable to check if the URL can be opened.

Reply:
URL and URI is not the same thing, here we are talking about an URI

I can update this to something like `URI not supported: ${targetUrl}` because we are able to check if the URI can be opened but it's just not supported or able to be opened